### PR TITLE
fix(rc): federation classified banner displayed after leaving or user deleted (AR-3264)

### PR DESCRIPTION
**Pull request by yamilmedina**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3264" title="AR-3264" target="_blank">BugAR-3264</a>  When I leave or get removed from a conversation the classified banner for that conversation is not displayed anymore
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

When leaving a conversation, or a user is blocked, or user deleted. The classified banner was not showing.

### Solutions

Display the component for other interactivity cases on the message composer

### Attachments (Optional)

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1629/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.0(6165489)(6165489)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.0(6165489)(6165489)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerClassifiedBanner.kt [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    paddingValues: PaddingValues = PaddingValues()
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .fillMaxWidth()
+            .background(color = colorsScheme().backgroundVariant)
+            .padding(paddingValues)
+    ) {
+        val isClassifiedConversation = securityClassificationType != SecurityClassificationType.NONE
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerClassifiedBanner.kt]: If the conversation classification type is set to `NONE` the `Row` element will still be there right? even though it doesn't have any height or anything that should be shown, maybe we could "remove" it completely if there is no need to show based on the classification type? let me know what you think

Comment (yamilmedina) [reply app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerClassifiedBanner.kt]: good eye! got catch while refactoring this component

Comment (yamilmedina) [reply app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerClassifiedBanner.kt]: I just moved the if block outside

**Review by alexandreferris — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1629/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.0(6165690)(6165690)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.0(6165690)(6165690)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerClassifiedBanner.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import com.wire.kalium.logic.feature.conversation.SecurityClassificationType
+
+@Composable
+fun MessageComposerClassifiedBanner(
+    securityClassificationType: SecurityClassificationType,
+    paddingValues: PaddingValues = PaddingValues()
+) {
+    val isClassifiedConversation = securityClassificationType != SecurityClassificationType.NONE
+    if (isClassifiedConversation) {
+        Row(
```
Comment (Garzas) [inline app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerClassifiedBanner.kt:44]: I think you can use here `Column` and remove `Box` under it 🤔

**Review by Garzas — APPROVED** [review]
🎸  Just small suggestion to composable structure

**Pull request merged**